### ISSUE TITLE
Evan Czaplicki - Let's be mainstream! User focused design in Elm

### ISSUE THREAD

**Issue by joshburgess** [ashleygwilliams/breakfast-repo#1]
[Evan Czaplicki - Let's be mainstream! User focused design in Elm](https://www.youtube.com/watch?v=oYk8CKH7OhE)

A really good talk on Elm, including what makes it unique, how to make it more appealing to the mainstream, etc.

Side note:

Evan was hired by Prezi and uses Prezi in this presentation (very effectively, I might add).

Interesting links about that: 

[Why Prezi is Investing in FRP / ELM?](https://prezi.com/qaklogolc1qz/why-prezi-is-investing-in-frp-elm/)
[Working on Elm Full-Time](http://elm-lang.org/blog/announce/elm-and-prezi)

**Comment by ashleygwilliams** [comment]
thanks for the suggestion! elm is p cool and i def want to know more about it :)

**Comment by joshburgess** [comment]
Of course! I peeped your website the other day after watching your Cows in Space/Carl Sagan/JS talk :P and saw something on there about you learning "reactive programming." So, I figured you might like this. I also thought you might appreciate how Evan stresses the importance of keeping the ideas simple to promote mainstream adoption since you are into teaching, presenting, etc....

**Comment by texastoland** [comment]
In my mind this is Evan's best talk. It's much more about combatting indoctrination through language design than necessarily about Elm. Elm itself is educational and relatively easy to pick up if you decide to but has a few properties I disagreed with.
